src/core/Track.cpp

```diff
@@ -1125,25 +1133,176 @@ void TrackContentObjectView::contextMenuEvent( QContextMenuEvent * cme )
 	if( fixedTCOs() == false )
 	{
 		contextMenu.addAction( embed::getIconPixmap( "cancel" ),
-					tr( "Delete (middle mousebutton)" ),
-						this, SLOT( remove() ) );
+					tr( individualTCO
+						? "Delete (middle mousebutton)"
+						: "Delete selection (middle mousebutton)" ),
+						[this](){ contextMenuAction( Remove ); } );
```

Comment:
This and all the other lambdas are indented as if they are part of the tr() function. This is confusing and should be de-dented.